Tick2q/simulate.fs

```diff
@@ -11,10 +13,10 @@
         
         /// The type of the update function used to determine a single variable's new
         /// value based on the old values of all the variables as in an environment.
-        /// This type is the type of the DSL functions that determine the logic implemented.
-        and Update = Environment -> Result<Wire,string>
+        /// This type is the type of the domain-specific language (DSL) functions that determine the logic implemented.
+        type Update = Environment -> Result<Wire,string>
```

Comment:
Wire is not part of the circular definition of `Update`, `Variable` and `Environment`. Minor correctness change. Happy to revert if unnecessary.